### ISSUE TITLE
SA1313 false positive for chained lambdas with discards

### ISSUE THREAD

**Issue by djfr** [DotNetAnalyzers/StyleCopAnalyzers#3017]
Tested with `StyleCop.Analyzers 1.1.118`

Reproduction:

```c#
var action = new Action<string, string>((_, __) =>
{
    var innerAction = new Action<string, string>((___, ____) => { }); // false SA1313
});
```

**Comment by sharwell** [comment]
This is covered by https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/1606#issuecomment-365563606 and https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/1606#issuecomment-415463606. 😄

**Comment by sharwell** [comment]
Duplicate of #1606
